dydx/dydxPresenters/dydxPresenters/_v4/KeyValueStoreProtocolStore+Ext.swift

```diff
@@ -22,14 +22,14 @@ public enum dydxSettingsStoreKey: String, CaseIterable {
         case .language: return DataLocalizer.shared?.language
         case .v4Theme: return dydxThemeType.classicDark.rawValue
         case .directionColorPreference: return "green_is_up"
-        case .shouldDisplayInAppNotifications: return true
+        case .shouldDisplayInAppNotifications: return false
```

Comment:
default is false here, but true in alerts worker dydx/dydxPresenters/dydxPresenters/_v4/GlobalWorkers/Workers/dydxAlertsWorker.swift

Unsure if the default would ever actually be used in `dydxAlertsWorker` though